### PR TITLE
APP-145 Android Sample App "City Guide"

### PR DESCRIPTION
**Pull request by maxml**
http://jira.kaaproject.org/browse/APP-145

Blocked by http://jira.kaaproject.org/browse/APP-148 - not integrated in sandbox.
Code is working, I am working on integration process.

Please, review code.

### CODE REVIEW

File: cityguide/source/app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -20,15 +19,21 @@
...
     <string name="app_name">City Guide Demo</string>
+
     <string name="action_set_location">Set location</string>
     <string name="select_area">Select area:</string>
     <string name="select_city">Select city:</string>
     <string name="areas_title">Areas</string>
     <string name="please_wait">Please wait...</string>
     <string name="show_on_map">Show on map</string>
+
+    <string name="no_areas">Kaa client is not started. Wait for it! We show your areas!</string>
```
Comment (ikulikov) [inline cityguide/source/app/src/main/res/values/strings.xml]: Areas info isn't available. The areas will be shown as soon as Kaa client will be started.

File: cityguide/source/app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -20,15 +19,21 @@
...
+    <string name="no_cities">Kaa client is not started. Wait for it! We show your cities!</string>
```
Comment (ikulikov) [inline cityguide/source/app/src/main/res/values/strings.xml]: Cities info isn't available. The cities will be shown as soon as Kaa client will be started.

File: cityguide/source/app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -20,15 +19,21 @@
...
+    <string name="no_city">Kaa client is not started. Wait for it! We show your city!</string>
```
Comment (ikulikov) [inline cityguide/source/app/src/main/res/values/strings.xml]: City info isn't available. The city details will be shown as soon as Kaa client will be started.

**Comment by vadimol** [comment]
please, resolve merge conflicts locally and update the PR

**Pull request merged**